src/ring_theory/derivation.lean

```diff
@@ -721,4 +731,74 @@ def kaehler_differential.linear_map_equiv_derivation : (Ω[S⁄R] →ₗ[S] M) 
   right_inv := derivation.lift_kaehler_differential_comp,
   ..(derivation.llcomp.flip $ kaehler_differential.D R S) }
 
+/-- The quotient ring of `S ⊗ S ⧸ J` by `Ω[S⁄R]` is isomorphic to `S`. -/
+def kaehler_differential.quotient_cotangent_ideal :
```

Comment:
Can you upgrade this to an iso of `S`-modules?

Reply:
Can you use `Ω[S⁄R]` in the statement? And isn't there a `^2` missing in the docstring?

Reply:
For the former: No, `Ω[S⁄R]` is not defined as an ideal of `S ⊗ S ⧸ J ^ 2`.
For the latter: Yes. Fixed.